scripts/prepdocs.py

```diff
@@ -40,6 +41,15 @@
 CACHE_KEY_CREATED_TIME = 'created_time'
 CACHE_KEY_TOKEN_TYPE = 'token_type'
 
+#Embedding batch section
+AOAI_EMBEDDING_MODEL = 'text-embedding-ada-002'
```

Comment:
I think this is passed via --openaideployment:

_parser.add_argument("--openaideployment", help="Name of the Azure OpenAI model deployment for an embedding model ('text-embedding-ada-002' recommended)")_

Reply:
That help text is confusing since thats actually the deployment name, not the model name.

You could add this to main.bicep:
output AZURE_OPENAI_EMB_MODEL string = embeddingModelName

And then add an openaimodel arg here and in the sh/ps1 script.

Reply:
done

Reply:
Looks like prepdocs.py takes this as an argument now, right?

Reply:
thats right, made changes to read it better.